tests/kafkatest/sanity_checks/test_performance_services.py

```diff
@@ -37,12 +37,12 @@ def setUp(self):
 
     # We are keeping 0.8.2 here so that we don't inadvertently break support for it. Since this is just a sanity check,
     # the overhead should be manageable.
-    @parametrize(version=str(LATEST_0_8_2))
+    @parametrize(version=str(LATEST_0_8_2), new_consumer=False)
     @parametrize(version=str(LATEST_0_9), new_consumer=False)
     @parametrize(version=str(LATEST_0_9), new_consumer=True)
```

Comment:
Thanks. I removed the redundant parameter. I noticed in other places in the code this redundancy does not exist.